app/upgrades/v5/upgrade.go

```diff
@@ -186,6 +192,25 @@ func updateGAMMParams(ctx sdk.Context, k *gammkeeper.Keeper) {
 	k.SetParams(ctx, params)
 }
 
+// addAuthorizedCircuitBreaker
+func addAuthorizedCircuitBreaker(ctx sdk.Context, k *circuitkeeper.Keeper, ak *authkeeper.AccountKeeper) {
+	permissions := circuittypes.Permissions{
+		Level: circuittypes.Permissions_LEVEL_ALL_MSGS,
```

Comment:
suggest changing to Super Admin permission so would be possible to give other addresses permissions to circuit break if we need.